Proofs/Experiments/AbsVCG.lean

```diff
@@ -274,6 +272,7 @@ theorem program_effects_lemma (h_pre : abs_pre s0)
             BitVec.truncate 64 1#32)) := by
     simp (config := {decide := true}) only [h_step_4, h_step_3, h_step_2, h_step_1,
                                             state_simp_rules]
+    rfl
```

Comment:
Not sure why this rfl is now needed, but the proof broke without it

Reply:
It's brittle anyway -- I'll redo all of it later. Thanks!